app/views/step_nav/show.html.erb

```diff
@@ -1,4 +1,14 @@
 <% content_for :title, step_by_step.title %>
+<% if step_by_step.recruitment_survey_url %>
```

Comment:
And we need something to tweak the spacing here too but I can't figure out how to do it!

Reply:
Ah, same as you've done in the smart answers PR with the css change 👍